dotnet/src/Extensions/Planning.ActionPlanner/skprompt.txt

```diff
@@ -0,0 +1,12 @@
+A planner takes a list of functions, a goal, and chooses which function to use.
+For each function the list includes details about the input parameters.
+{{basicPlanner.GoodExamples}}
+{{basicPlanner.EdgeCaseExamples}}
+[END OF EXAMPLES]
```

Comment:
Are there always some examples? Or could this [END] be left hanging?

Reply:
in my experience this is clearer and works well with old models, ie clearly marking each section start and end by name